src/sandesh/Assignment_15/PersonalBankAccountManager.java

```diff
@@ -0,0 +1,295 @@
+/*Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+.
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI*/
+
+package sandesh.Assignment_15;
+
+import java.util.Scanner;
+
+public class PersonalBankAccountManager {
+
+	static double hdfcBalance, iciciBalance, bobBalance, sbiBalance;
+	static int hdfcCreditOperations, hdfcDebitOperations, iciciCreditOperations, iciciDebitOperations;
+	static int bobCreditOperations, bobDebitOperations, sbiCreditOperations, sbiDebitOperations;
+
+	double performCreditDebit(String operationType, double amount, String bankName) {
```

Comment:
You should use switch case instead of this if else statements, it give good readability.

instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

Reply:
sure @harshvegada will keep these points in mind and work on the same. Thank you :)